### DISCUSSION TITLE
I can't create an education account.

### DISCUSSION THREAD

**Discussion by middle1** [community/community#128517] [GitHub Education] [answered]
### Hi everyone,

I tried to create an account, but an error occurred at the geolocation confirmation stage, which prevented me from continuing to create an account.
Error: "error we require applicants to use a school-issued email address to apply"

**Comment by davevad93** [comment]
Hi @middle1 ,

That means you need a verified school-issued email address linked to your GitHub account, the requirements to be eligible for GitHub Education are the following, as stated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student):

- Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution
- Have a verifiable school-issued email address or upload documents that prove your current student status
- Have a [GitHub personal account](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github)
- Be at least 13 years old

**Comment by samus-aran** [comment, marked as answer]
Hi @middle1 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I don't have a school-issued email address, what other official proof of enrollment is accepted?](https://github.com/orgs/community/discussions/111352#i-dont-have-a-school-issued-email-address)
- [What types of evidence does the Student Developer Pack application accept?](https://github.com/orgs/community/discussions/111352#what-types-of-evidence-does-the-student-developer-pack-application-accept)
- [My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?](https://github.com/orgs/community/discussions/111352#selected-school-incorrect-details)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

**Reply by middle1** [reply]
Thanks
